backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5617260_sys_gh12099_Sales_order_window_adjustments.sql

```diff
@@ -0,0 +1,30 @@
+-- 2021-12-06T15:10:06.746Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_ColumnCallout (AD_Client_ID,AD_ColumnCallout_ID,AD_Column_ID,AD_Org_ID,AD_Table_ID,Classname,Created,CreatedBy,EntityType,IsActive,SeqNo,Updated,UpdatedBy) VALUES (0,541246,2197,0,259,'org.compiere.model.CalloutOrder.shipper',TO_TIMESTAMP('2021-12-06 16:10:06','YYYY-MM-DD HH24:MI:SS'),100,'D','Y',10,TO_TIMESTAMP('2021-12-06 16:10:06','YYYY-MM-DD HH24:MI:SS'),100)
+;
```

Comment:
```suggestion
```

registering the callout in the DB is legacy; see the comments below on what to do instead